deployment/arm/deploy-global.sh

```diff
@@ -52,11 +65,13 @@ backendHosts=$(toArmArray ${backendHostArray[*]})
 echo "Creating global resource Group: $resourceGroupName"
 az group create \
   --name $resourceGroupName \
-  --location centralus
+  --location $primaryRegion
 
 echo "Deploying global resources to $resourceGroupName"
 az group deployment create \
   --name "Urlist-global-$(timestamp)" \
   --resource-group $resourceGroupName \
   --template-file global.json \
-  --parameters frontDoorName=$frontDoorName frontendHosts=$frontendHosts backendHosts=$backendHosts
+  --parameters frontDoorName=$frontDoorName frontendHosts=$frontendHosts backendHosts=$backendHosts \
+      cosmosdbName=$cosmosdbName cosmosdbPrimaryRegion=$primaryRegion \
+      cosmosdbSecondaryRegion=$secondaryRegion cosmosdbTertiaryRegion=$tertiaryRegion
```

Comment:
Is there a way to accept a single argument like `cosmosRegions` vs explicitly settings `primary`, `secondary`, etc.?

Reply:
Thats a good feedback. I`m looking into this one to make it dynamic, that way we wont have to hard code number of regions.

Reply:
I have added the cosmosRegions argument and deleted primary, etc. Its much cleaner that way.